src/schema/me/order/index.js

```diff
@@ -0,0 +1,112 @@
+import {
+  GraphQLInt,
+  GraphQLList,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import Artwork from "schema/artwork"
+import EditionSet from "schema/edition_set"
+import Location from "schema/location"
+import Partner from "schema/partner"
+import money from "schema/fields/money"
+import { GravityIDFields } from "schema/object_identification"
+
+export const OrderLineItemType = new GraphQLObjectType({
+  name: "OrderLineItem",
+  fields: () => ({
+    ...GravityIDFields,
+    quantity: {
+      type: GraphQLInt,
+      description: "Quantity of this item",
+    },
+    artwork: {
+      type: Artwork.type,
+      description: "Artwork that is being ordered",
+    },
+    edition_set: {
+      type: EditionSet.type,
+      description: "Edition set on the artwork",
+      resolve: ({ edition_set }) => edition_set,
+    },
+    partner: {
+      type: Partner.type,
+      description: "Partner being ordered from",
+      resolve: ({ partner }) => partner,
+    },
+    partner_location: {
+      type: Location.type,
+      description: "Location of the partner",
+      resolve: ({ partner_location }) => partner_location,
+    },
+    shipping_note: {
+      type: GraphQLString,
+      description: "Shipping note from the partner",
+    },
+    sale_conditions_url: {
+      type: GraphQLString,
+      description: "Sale conditions (set by partner)",
+    },
+  }),
+})
+
+export const AddressType = new GraphQLObjectType({
+  name: "Address",
+  fields: () => ({
+    ...GravityIDFields,
```

Comment:
The `GravityIDFields` implies that this is an actual type from Gravity (which must include a non-null `_id`and `id`), maybe `Address` can live as a generic interface w/o these, and then an `OrderAddress` which implements and mixes in the ids?

Just spit-balling for future-proofing schema (when there's another microservice that wants to expose an 'address').

Reply:
That makes sense, but these addresses actually are stored in gravity (and thus have `id`s) and the model is pretty tied to the fields that are available there. 

In any case I'd rather leave this for now and extend it/abstract it when the need arises!